.github/scripts/run-nightly-nodocker.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+set -euo pipefail
+
+# Sanity checks
+if [[ ! -v GITHUB_RUN_ID ]]; then
+    echo "GITHUB_RUN_ID is not set! Please check your environment variable. Stop."
+    exit 1
+fi
+
+if [[ ! -v SCRIBE_GRAPHQL_ACCESS_TOKEN ]]; then
+    echo "SCRIBE_GRAPHQL_ACCESS_TOKEN is not set! Please check your environment variable. Stop."
+    exit 1
+fi
+
+# Version of the config
+CONFIG_VER=v0
+CONFIG_DIR=${PWD}/torchbenchmark/score/configs/${CONFIG_VER}
+CONFIG_ENV=${CONFIG_DIR}/config-${CONFIG_VER}.env
+CONDA_ENV_NAME=gh${GITHUB_RUN_ID}
+DATA_DIR=${HOME}/benchmark-results-v0.1/gh${GITHUB_RUN_ID}
+
+# Load environment variables
+set -a;
+source ${CONFIG_ENV}
+set +a;
+DATA_JSON_PREFIX=$(date +"%Y%m%d_%H%M%S")
+sudo nvidia-smi -ac ${GPU_FREQUENCY}
+export CUDA_VISIBLE_DEVICES="${GPU_LIST}"
+export GOMP_CPU_AFFINITY="${CORE_LIST}"
+
+mkdir -p ${DATA_DIR}
+conda create -y -q --name ${CONDA_ENV_NAME} python=${PYTHON_VERSION}
+. activate ${CONDA_ENV_NAME}
+conda init bash; conda run /bin/bash
+
+cp docker/multi30k.tar.gz torchbenchmark/models/attention_is_all_you_need_pytorch
+pushd torchbenchmark/models/attention_is_all_you_need_pytorch
+mkdir -p .data/multi30k
```

Comment:
these are some kind of caching to speed up the subsequent install.py?

Reply:
This is because multi30k.tar.gz is not consistently available (the download fails sometimes). To make sure CI is not interrupted by the broken URL link, I store it locally in the repository.

Reply:
Previously we didn't have this problem because the files was packaged together in the Docker, see: https://github.com/pytorch/benchmark/pull/178

Reply:
ok, yea this is a good catch. those download mirrors are annoying.